src/components/Button/Button.scss

```diff
@@ -0,0 +1,39 @@
+.button {
+  justify-self: center;
+  border-radius: 5px;
+  border-width: 0px;
+  padding: 20px;
+  font-weight: bold;
+  background-repeat: no-repeat;
+  background-position: center;
+  color: #EC823E;
+  background-color: #F7F8FD;
+  height: 35px;
+
+  &--orange{
+    color: #FFE3BB;
+    background-color: #EC823E;
+  }
+     
+  &__icon{
+  
+    &--plus{
+      background-image: url('../../images/icon-plus.svg');
+    }
+  
+    &--minus {
+      background-image: url('../../images/icon-minus.svg');
+    }
+  
+    &--add { 
+      line-height: 15px;
+      padding-left: 60px;
+      padding-right: 35px;

```

Comment:
What if we move this one to the  `.button {}` rules? And here we just duplicate the value of the padding-left + icon + margin-right icon